www/templates/www/user.html

```diff
@@ -7,13 +7,6 @@ <h1>
     {% trans 'User data for' %} {{ userdetails.first_name }} {{ userdetails.last_name }}
   </h1>
 
-  {% if user.is_staff %}
-    <p>
-      <a href="{% url 'updateuser' userdetails.id %}">{% trans "Update user's data" %}</a> |
-      <a href="{% url 'admin:users_customuser_change' userdetails.id %}">{% trans "View in admin" %}</a>
-    </p>
-  {% endif %}
-
```

Comment:
I'm not too sure removing "quick link at admin" from users side for admins really helps that much, usually admin a re on user view page anyway to see what is wrong already, thus link should exist there (assuming django-admin edit place is needed at all, which it oftentimes is). Update user data could be removed if F5 is same thing or whole fetch generally is nothing but refresh.

Reply:
With this change the user view looks the same for staff and regular user, which helps when talking with users (you see the same thing they see)

The quick admin link is still available in users listing page (the end goal is to remove the admin part completely by having all the necessary functions in the www ui)